### PR TITLE
Reduce heap size for Elastisearch

### LINKED ISSUE #1225: Deployment of development branch to Herokuapp not working
The development branch is deployed to https://loklak-server-dev.herokuapp.com

Currently it shows an application error.

### PR DESCRIPTION
**Pull request by singhpratyush**
### Short description

Fixes #1225.

The max allowable RAM usage for single heroku dyno is 512 MB (for free one) and the app is killed upon exceeding 200% usage (including swap). So, reducing the max allowable memory will stop the app from crashing.

This PR sets the environment variable `ES_HEAP_SIZE` to 100m so that it doesn't utilise a lot of memory.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: bin/heroku_start.sh [hunk 1]
```diff
@@ -3,4 +3,7 @@
 # Make sure we're on project root
 cd $(dirname $0)/..
 
+# Reduce Xmx to 400 MB
+sed -i 's/^\(Xmx=\).*/\1400m/' conf/config.properties
```
Comment (vibhcool) [inline bin/heroku_start.sh]: setting variable and then using it in this line will be easier to modify later

Comment (singhpratyush) [inline bin/heroku_start.sh]: @vibhcool: I have fixed this.

**Comment by singhpratyush** [comment]
Yes. The app still crashes because ES is taking up a lot of memory. I have made some changes and monitoring the status. Will ping when I find something.

**Comment by mariobehling** [comment]
@singhpratyush How can you speed up the process to merge this PR?

**Comment by singhpratyush** [comment]
@mariobehling @vibhcool: This would be a good configuration for the heroku deployment. But this still wouldn't be stable.

512 MB of memory is really less to run loklak. What should we do about that?

**Comment by singhpratyush** [comment]
@Achint08 @djmgit @hemantjadon @kavithaenair @SarishiNoHara @SKrPl @vibhcool: Please review.

**Comment by hemantjadon** [comment]
@singhpratyush @mariobehling This configuration seems decent, but the question is can Loklak work on low memory? IMO we will have to make the upgrade sometime soon

**Comment by sudheesh001** [comment]
I don't see a major problem in merging this patch for heroku. The heroku script is only used on heroku and can run the build. Can you please update your branch and provide a test link so that we can test if it's still working.

I see mentions in the gitter chat that the loklak server is becoming unstable with a lower configuration, can someone please explain this.

**Comment by mariobehling** [comment]
> This configuration seems decent, but the question is can Loklak work on low memory? 

We need to try that out. I saw that loklak usually take 1-1.5 GB.

**Comment by singhpratyush** [comment]
@sudheesh001: The free heroku deployment offers 512m of memory and extra 512m as swap. It shuts down an application after it has utilised all of it (512m + 512m).

And loklak happens to overshoot that number very fast. I tried to restrict the `Xmx` and ES heap but the deployment still keeps on restarting.

**Pull request closed without merging**